recipes/miktex-portable/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "2.9.5857" %}
+
+package:
+  name: miktex-portable
+  version: {{ version }}
+
+source:
+  fn: miktex-portable-{{ version }}.exe
+  url: http://mirrors.ctan.org/systems/win32/miktex/setup/miktex-portable-{{ version }}.exe
+  sha1: 2b50fd29b3683dbe769dd1891e075ef8941658f7
+
+build:
+  skip: True  # [not win]
+  number: 4
+  binary_relocation: false
```

Comment:
I got that from the original packages. As far as I understand it, it is not needed as the miktex packages is already portable and so the binaries do not need to be checked for absolute paths.